include/glow/Base/Tensor.h

```diff
@@ -629,6 +643,16 @@ class Tensor final {
     }
     return numExceedingError == 0;
   }
+
+  bool isBitwiseEqualImpl(const Tensor &other) const {
+    auto const *myData = getRawDataPointer<uint8_t>();
```

Comment:
I think you need: `getUnsafePtr`

Reply:
Correct.  Thanks, Roman.  I missed this because I was testing on a release build (without asserts).